docs/UserGuide.md

```diff
@@ -32,6 +32,7 @@ Throughout the document you may encounter some of these symbols, and they denote
     * [Save recipes](#save-recipes)
     * [Reset to the default recipes: `reset`](#reset-to-the-default-recipes-reset)
     * [View an existing recipe: `view`](#view-an-existing-recipe-view)
+    * [Exiting the program: `Exit`](#exiting-the-program)
```

Comment:
just one small nit, we should move this line before the `Find an existing recipe`, since TOC is sorted in alphabetical order.